### DISCUSSION TITLE
wxWidgets how to add webview and webview2

### DISCUSSION THREAD

**Discussion by Jono-Moss** [microsoft/vcpkg#27981] [Q&A] [answered]
Good Day,

I get wxWidgets via the following:
`vcpkg install wxwidgets:x64-windows-static`
But in my cmake project it is missing the webview library.

```
CMake Error at C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find wxWidgets (missing: webview) (found suitable version
  "3.2.1", minimum required is "3.1")
```
And I don't see it anywhere in the vcpkg\installed\x64-windows-static folder.
How would I go about adding the webview and webview2 (Edge) libraries using vcpkg.

Thank you.

**Comment by dg0yt** [comment, marked as answer]
`webview` is an optional feature in port wxwidgets. You must do
~~~
vcpkg install wxwidgets[webview]:x64-windows-static

**Reply by Jono-Moss** [reply]
Wow thank you so so much for your help. This helps me so much.
